### ISSUE TITLE
Feature request: No shortcut

### ISSUE THREAD

**Issue by ErikFontanel** [exevil/Keys-For-Sketch#7]
Hi Vyacheslav,

Thank you for this great plugin, love the integration with the preferences pane. I use quite some plugins and Sketch Runner. Some plugins come with default shortcuts. Some of these clash with my own defined shortcuts. Which is very annoying because most plugin actions are so uncommonly used, they don't even need a shortcut. Especially when combined with Sketch Runner.

With [Sketch Plugin Monster](https://github.com/PeachScript/sketch-plugin-monster) it is possible to set a blank shortcut btw, but the Sketch integration isn't so nice.

**Comment by exevil** [comment]
Hey, Erik! Thanks for the feedback.

TL;DR: I'm sorry but I'm not planning to add the requested functionality to Keys. 😐

To understand why you can imagine that any custom shortcut set through Keys is exactly the same thing as a custom shortcut that set through the System Preferences. So while you can't assign an empty shortcut there, you can't do it in Keys as well.

Of course, I can (and I've actually done in early stages of development) implement some local shortcut management system which is able to assign empty shortcuts and a couple of other tricky things, but here're some advantages that wrapping of the macOS shortcut management provides to end user:
1. If you previously set your custom shortcuts via the System Preferences, it becomes manageable by Keys out of the box right after a plugin installation;
2. If I discontinue the plugin support and/or a new version of Sketch will mess up APIs that Keys used to inject and manipulate itself, all your custom shortcuts will continue to work as before. Moreover, it still remains editable through the System Preferences.

Hopefully, I explained my position clearly.

Thanks again for the feedback.
Cheers! ✌️

**Comment by ErikFontanel** [comment]
Loud and clear! I guess I will make up some bogus shortcuts then.
